packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -126,13 +166,35 @@ export default class QuanticSearchBox extends LightningElement {
     }
     this.state = this.searchBox?.state;
     this.suggestions =
-      this.state?.suggestions?.map((s, index) => ({
+      this.state?.suggestions?.map((suggestion, index) => ({
         key: index,
-        rawValue: s.rawValue,
-        value: s.highlightedValue,
+        rawValue: suggestion.rawValue,
+        value: suggestion.highlightedValue,
       })) ?? [];
   }
 
+  updateRecentQueriesListState() {
+    if (this.recentQueriesList.state?.queries) {
+      this.recentQueries = this.recentQueriesList.state.queries;
+      setItemInLocalStorage(
+        this.localStorageKey,
+        this.recentQueriesList.state.queries
+      );
+    }
+  }
```

Comment:
```suggestion
updateRecentQueriesListState() {
  const { recentQueries } = this.recentQueriesList.state || {};

  if (recentQueries) {
    this.recentQueries = recentQueries;
    setItemInLocalStorage(this.localStorageKey, recentQueries);
  }
}
```

what do you think of this instead?

Reply:
you probably meant to write `queries` instead of `recentQueries`.
 I'm not really a fan of trying to destructure  a property from a possibly empty object `{}`.

This suggestion could be better, but it's very minor, it's just a coding style preference that does not necessarily have an impact

```
  updateRecentQueriesListState() {
    if (this.recentQueriesList.state?.queries) {
      const {queries} = this.recentQueriesList.state;
      this.recentQueries = queries;
      setItemInLocalStorage(this.localStorageKey, queries);
    }
  }
```